.github/scripts/utils.zsh/check_linux

```diff
@@ -1,4 +1,19 @@
-autoload -Uz log_info log_status log_error log_debug log_warning
+autoload -Uz log_info log_status log_error log_debug log_warning log_group
+
+log_group 'Check Linux build requirements'
+log_debug 'Checking Linux distribution name and version...'
+
+typeset -g -i UBUNTU_2210_OR_LATER=0
```

Comment:
Remind me what the significance of UBUNTU_2210_OR_LATER is? Some specific feature we need to detect?

Reply:
`libsrt` and `librist` are both available in Ubuntu 22.10, but not 22.04. As the file is a straight clone of the variant used for obs-studio in the future, it was left in to make updates/staying in sync with the main repo easier.

Reply:
Could a comment to that effect be left here? I was struggling to recall why this specific version was significant.

Reply:
Added comment.